packages/monorepo-scripts/README.md

```diff
@@ -2,6 +2,16 @@
 
 This repository contains a few helpful scripts for working with this mono repo.
 
+#### Scripts
+
+**`yarn deps_versions`**: Since we use Lerna + yarn workspaces, shared dependencies between packages in the monorepo get hoisted to a top-level `node_modules` directory. If two packages use different versions of the same dependency however, both get installed. To avoid having many versions of a dependency installed, we try to keep dependency versions the same across packages in the monorepo. This script will list any dependencies for which we have multiple versions installed. We can then go through them and try to consolidate to a single version where possible.
```

Comment:
nit: should we capitalize "yarn" as well? just looking at the yarn website looks like they capitalize it